### ISSUE TITLE
Notify the user 3 days before servers are suspended

### ISSUE THREAD

**Issue by anondev-sudo** [Ctrlpanel-gg/panel#255]
# 🚀 Feature Request

### Is your proposal related to a problem?

*Provide a clear and concise description of what the problem is.
For example, "I can't change x and it makes me have to do manual work"*

Users don't check dashboard often and then they are confused when their servers were suspended.

### Describe the solution you'd like

*Provide a clear and concise description of what you want to happen.
For Example, "Automate the changing of x so I don't have to do it manually"*

Notify the user 3 days before their servers will be suspended. Send them email.

### Additional context

*Is there anything else you can add about the proposal?
You might want to link to related issues here, if you haven't already.*

Maybe make it optional?

**Comment by anondev-sudo** [comment]
Make it as config option for admins that they can change days before suspend reminder, when they set it to 0, it should be disabled.

**Comment by anondev-sudo** [comment]
When someone will look at it, you should also consider adding that notification that servers were suspended will be send just once - #246

Also this reminder before suspension should be send once.

**Comment by S0ly** [comment]
have this been done ?

**Comment by S0ly** [comment]
@IceToast it was you on the new billing right ?

**Comment by IceToast** [comment]
> @IceToast it was you on the new billing right ?

Yes, cant remember if this has been changed too, its not that high of a priority

**Comment by S0ly** [comment]
Yes but would be nice to check to close useless issues

**Comment by MrWeez** [comment]
Implemented in 1.2.0 release
